docker-compose.yaml

```diff
@@ -29,8 +29,8 @@ services:
     build: .
     volumes:
      - .:/dockermount
-    command: >
-      /bin/sh -c "mkdir $$MATVIEW_SQL_DIR &&
+    command:
+      /bin/sh -c "mkdir $$MATVIEW_SQL_DIR | true &&
```

Comment:
Is this left over from the docker-compose stuff or did the changes to the matviews require this to be changed also?

Reply:
This is a more compact fix to the original issue suggested to me by will. It is not strictly nessicary for the matview changes to work.